official-antistasi-community/A3-Antistasi
# Facewear support for AI loadouts

## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [X] Enhancement

### What have you changed and why?
This pull request adds the ability for AI to randomly select from facewear items to wear specified in their template (e.g. goggles, glasses, bandanas, custom mod items like beards, etc.).

![image](https://user-images.githubusercontent.com/92134872/176320127-6be9fe66-2dd4-4e1c-9636-ba2219f641a3.png)


This has been something requested occasionally by other community members on the Discord server, and I personally wanted to add this functionality myself as well after I started playing around with faction templates. Essentially, its overall purpose is adding more variety with AI appearance.

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [X] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
The AAF and FIA factions are setup with facewear. By spawning any FIA units (except for Petros), or by travelling to an area with AAF units, you can observe the randomized facewear on the AI.
